### DISCUSSION TITLE
how disable retry send mail

### DISCUSSION THREAD

**Discussion by aminkhan20** [postalserver/postal#2074] [Help with using Postal] [answered]
Hello
When an invalid email is sent to the Gmail servers, it is marked with the softfail tag
How can we disable this retry or reduce it to at least 2 times?

**Comment by willpower232** [comment, marked as answer]
I think you can control the retries in general with https://github.com/postalserver/postal/blob/9f0697f194209f5fae5e451ba8fb888413fe37fa/config/postal.defaults.yml#L14

But if you're looking for something specific to this situation, I don't think there is a solution other than to not send the message in the first place.

**Reply by aminkhan20** [reply]
Thank you for your response
